tools/check_test.js

```diff
@@ -0,0 +1,170 @@
+/* Copyright 2016 Samsung Electronics Co., Ltd.
```

Comment:
Regarding this opinion, I don't totally agree, ``test`` is the directory which are for testfiles literally, and ``tools`` is the directory which are tools for utility such as check license, or run tests. Can we just move ``test`` and ``tools`` both? I don't think that is a big burden?